Frontend/ui-customer/src/Screens/Types.purs

```diff
@@ -862,6 +863,7 @@ type Fares = {
 , pickupCharges :: String
 , nominalFare :: String
 , waitingCharges :: String
+, referenceString :: String
```

Comment:
why is this required here . it is not used anywhere and we are going to deprecate this `Fares` type so add it separate